_extensions/seedcase-theme/theme.scss

```diff
@@ -1,13 +1,20 @@
 /*-- scss:defaults --*/
 
-@import url('https://fonts.googleapis.com/css2?family=Poppins');
-@import url('https://fonts.googleapis.com/css2?family=Lato');
+@import url('https://fonts.googleapis.com/css2?family=Noto Sans');
 @import url('https://fonts.googleapis.com/css2?family=Fira+Code');
+@import url('https://fonts.googleapis.com/css2?family=Maven+Pro');
 
-$font-family-sans-serif: "Lato", sans-serif;
+$font-family-sans-serif: "Maven Pro", sans-serif;
 $font-family-monospace: "Fira Code", monospace;
-$headings-font-family: "Poppins", sans-serif;
-$navbar-bg: #990032;
+$headings-font-family: "Noto Sans", sans-serif;
+
+/* brand colour #003224 - the one below is a slightly lighter version of the brand colour */
+$primary: #196440;
+$secondary: #C6F0D3;
+$link-color: #8b8b8b;
```

Comment:
@signekb it is this colour

Reply:
Wait, the #8b8b8b? That's grey, right?

Reply:
That's what I mean! It's a terrible colour for colour blind, I can't tell what it is!!

Reply:
Ah, alright! Definitely! I was also unsure of why you had placed the grey as the "link colour"